trio/_highlevel_open_tcp_listeners.py

```diff
@@ -121,9 +121,10 @@ def _compute_backlog(backlog):
             try:
                 # See https://github.com/python-trio/trio/issues/39
                 if sys.platform == "win32":
-                    sock.setsockopt(
-                        tsocket.SOL_SOCKET, tsocket.SO_EXCLUSIVEADDRUSE, 1
-                    )
+                    # See https://github.com/python-trio/trio/issues/928
+                    # We used to set SO_EXCLUSIVEADDRUSE here, but determined
+                    # that we shouldn't.
+                    pass
```

Comment:
Given that the conclusion is `SO_EXCLUSIVEADDRUSE` is kinda useless and no-one cares about it, I think we can drop all mention of it from the code. (The issue tracker, git history, and release notes will still be there to remind us if we ever need to look something up :-).) So I'd collapse the whole if/else down to just:

```python
if sys.platform != "win32":
    sock.setsockopt(tsocket.SOL_SOCKET, tsocket.SO_REUSEADDR, 1)
```

Reply:
Thanks